styles/utility.css

```diff
@@ -4,6 +4,10 @@
     background: var(--indigo);
```

Comment:
Don't use direct and only colour names for colour variables, that can7be changed